safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -99,7 +120,7 @@ def on_output_path_textChanged(self):
     def on_input_path_textChanged(self):
         """Action when input file name is changed."""
         input_path = self.input_path.text()
-        input_not_grid_msg = self.tr('input file is not .xml')
+        input_not_grid_msg = tr('input file is not .xml')
```

Comment:
Should we do a search and replace in the code for `self.tr(` to `tr(` ?

Reply:
Yeah, I think we can do that. It should be easy I think.